src/FileIO/BasicIOHelper/FileLoader.java

```diff
@@ -0,0 +1,79 @@
+package FileIO.BasicIOHelper;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.LinkedList;
+import java.io.File;
+
+/**
+ * A simple, easy-to-use class that reads and writes from a file
+ * 
+ * @author David Harmeyer
+ */
+public class FileLoader {
```

Comment:
Name FileLoader is not accurate, since this class reads and writes to a file, change it to something more descriptive.